runtime/metrics_api.go

```diff
@@ -145,7 +145,7 @@ func (m *MetricsApi) RecordPoint(name string, value float64) {
 }
 
 func (m *MetricsApi) Counter(name string) Counter {
-	return m.reg.GetOrRegister(name, metrics.NewCounter()).(metrics.Counter)
+	return m.reg.GetOrRegister(name, NewResettingCounter()).(metrics.Counter)
```

Comment:
@raulk I think by default it makes it easier to use the resetting counter, than the counter. It resets every time we emit it (i.e. when calling `Snapshot`).

The difference is the following:

Resetting counter:
<img width="739" alt="Screenshot 2020-05-13 at 15 28 09" src="https://user-images.githubusercontent.com/50459/81818592-845dde80-952e-11ea-8b27-e249a784879d.png">

Default counter:
<img width="739" alt="Screenshot 2020-05-13 at 15 29 00" src="https://user-images.githubusercontent.com/50459/81818621-8de74680-952e-11ea-8c92-ce1c11f7b492.png">

The counter you are looking at is 50 instances increasing a counter 5 times (50 instances starting 5 listeners).

The second screenshot is wrong, the sum total value has to be 250, but since this is an accumulating counter, we need to add some kind of query to subtract adjacent values. It is much easier to use a statsd counter type (rather than prometheus type) and just reset it when emitting it. Obviously with the default counter we can't use the `sum` aggregation in InfluxDB/Grafana (hence the wrong results).